### DISCUSSION TITLE
solidjs implementation

### DISCUSSION THREAD

**Discussion by edgarjoao** [patternfly/patternfly#5539] [General]
Hi PatternFly team,
Are you planning to implement PatternFly components with the solidjs in a future?

Thanks,
Edgar

**Comment by nicolethoen** [comment]
Not at this time.
Yours might be the first request for it.
The only JS implementation the PatternFly team builds/maintains is React.
There are a few community projects for other building out PF components using other frameworks like [Yew](https://github.com/ctron/patternfly-yew) and [Kotlin](https://github.com/patternfly-kotlin/patternfly-kotlin).

**Comment by edgarjoao** [comment]
Thank you @nicolethoen I'll take a look on the react version.
